src/logicblocks/event/types/projection.py

```diff
@@ -4,12 +4,10 @@
 from typing import Any, cast
 
 from logicblocks.event.types import (
-    CategoryIdentifier,
-    LogIdentifier,
-    StreamIdentifier,
+    EventSourceIdentifier,
 )
 
-type Projectable = LogIdentifier | CategoryIdentifier | StreamIdentifier
+type Projectable = EventSourceIdentifier
```

Comment:
I think this is fine for now but do you think it makes sense to allow projections across any `EventSource` in general? Considering that for example an entire Kafka topic could be considered an EventSource even though you'd only be allocated zero or more partitions of it.

Reply:
I'm not really sure to be honest with you. Probably not? Maybe we can mark the sources as projectable in some way?